GitTfs/Core/GitTfsRemote.cs

```diff
@@ -470,16 +470,13 @@ private string ProcessChangeset(ITfsChangeset changeset, LogEntry log)
                 }
 
                 if (!string.IsNullOrWhiteSpace(changeset.Summary.PolicyOverrideComment))
-                    log.Log += "\n" + GitTfsConstants.GitTfsPolicyOverrideCommentPrefix + changeset.Summary.PolicyOverrideComment;
+                    log.Log += $"\n{GitTfsConstants.GitTfsPolicyOverrideCommentPrefix} {changeset.Summary.PolicyOverrideComment}";
```

Comment:
When we won't support C# 6.0 then we need to change the 'C# Language Level' property on the projects to 'C# 5.0'.